srcpkgs/iptables/files/ip6tables/run

```
@@ -1,4 +1,4 @@
 #!/bin/sh
 [ ! -e /etc/iptables/ip6tables.rules ] && exit 0
-ip6tables-restore /etc/iptables/ip6tables.rules
+ip6tables-restore /etc/iptables/ip6tables.rules -w
```

What do you think about using `|| exit 1` instead?

This way the process cannot silently fail if the lock is acquired indefinitly.

---

Do you mean together with the `-w` like that:

```
ip6tables-restore /etc/iptables/ip6tables.rules -w $TIMEOUT || exit 1
```

With `TIMEOUT` defined in `conf` or with a reasonable value such as 3. Without any option it will wait indefinitely for the lock.

Or just `|| exit 1` and rely on `runsv` to restart the service?